### DISCUSSION TITLE
blog/ai-and-future

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#9] [Announcements]
# blog/ai-and-future

面对未来人工智能取代人类的问题，放弃人的视角不但幼稚，而且危险。

https://rercel.com/blog/ai-and-future

**Comment by darmau** [comment]
博主你好！
你的博客被我收录进了我的个人项目积薪(firewood.news)，这是一个独立博客收录导航站。为的是给如今还在坚持长文写作的博主提供一个流量来源。
目前这个站还在内测开发中。你的博客是我最早添加的一批，是我认为要么在内容上要么在设计上有不错表现的网站。
我建了一个微信群，邀请这些被收录的博客的站长加入，希望能从你这里获得第一手的反馈信息，进行后续的迭代优化。另外也想认识一些在这个时代还在坚持写作的人。
我看你之前发过放弃独立博客的文章，还好没有。
你给我发邮件（积薪-关于-提交博客），我给你发送二维码。
博客收录还在进行中。网站近期内会开始对外推广发布。
如果你不想入群也没关系。这是我这个项目的对外联系邮箱。你对这个网站有任何想法，欢迎通过邮件和我联系。

**Reply by Pathsis** [reply]
感谢关注！firewood.news做的非常漂亮，很荣幸被收录。
